### PR TITLE
Toni development

### PR DESCRIPTION
**Pull request by tncarr101**

### CODE REVIEW

File: walkthroughs/week-2-web-development/portfolio-walkthrough.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Engine to deploy that webpage to a public URL.
 You can return to this walkthrough anytime by running this command:
 
 ```bash
-teachme ~/step/walkthroughs/week-2-web-development/portfolio-walkthrough.md
+teachme ~/step/walkthroughs/week-2-web-development/portfolio-walkthrough.md`
```
Comment (qirongwang) [inline walkthroughs/week-2-web-development/portfolio-walkthrough.md]: is this change needed?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -59,4 +59,12 @@ function showSlides() {
   slides[slideIndex-1].style.display = "block";  
   dots[slideIndex-1].className += " active";
   setTimeout(showSlides, 2000); // Change image every 2 seconds
-}
\ No newline at end of file
+}
+  //fetch function
+  async function getRandomQuoteUsingAsyncAwait() {
+  const response = await fetch('/data');
```
Comment (qirongwang) [inline portfolio/src/main/webapp/script.js:66]: indent

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -19,14 +19,65 @@
...
+    //Creating an entity
+    Entity taskEntity = new Entity("Task");
+    taskEntity.setProperty("comments", commentsArr);
+
+    //Storing the Entity in datastore
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(taskEntity);
+}
+
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (qirongwang) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:55]: is this empty function needed?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -19,14 +19,65 @@
...
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Toni!</h1>");
+    //Initializing and adding in arrayList
+
+    private ArrayList<String> commentsArr;
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    commentsArr = new ArrayList<String>();
+    commentsArr.add(request.getParameter("fname"));
```
Comment (qirongwang) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: would be better to store these parameter names in constant

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -5,8 +5,13 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
   </head>
+<<<<<<< HEAD
```
Comment (qirongwang) [inline portfolio/src/main/webapp/index.html:8]: merge conflict checked in accidentally?

File: portfolio/pom.xml [hunk 6]
```diff
@@ -23,8 +23,26 @@
...
+  <version>1.9.59</version>
+</dependency>
+
+
   </dependencies>
 
+  
+
+
+
```
Comment (shuangjiec) [inline portfolio/pom.xml:45]: remove extra lines

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 7]
```diff
@@ -19,14 +19,65 @@
...
+    taskEntity.setProperty("comments", commentsArr);
+
+    //Storing the Entity in datastore
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(taskEntity);
+}
+
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  
+    }
```
Comment (shuangjiec) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:57]: remove unused code

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 8]
```diff
@@ -19,14 +19,65 @@
...
+
+
+
+
+ 
+
+
+
+
+
```
Comment (shuangjiec) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:82]: remove extra lines

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 3 above)
Comment (tncarr101) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:55]: When I try to delete it, I get errors. So that's I was keeping it there because it wasn't causing any errors.

Comment (shuangjiec) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:55]: Hey Toni, What error are you getting when you remove this method? (I couldn't find my previous comment and reply)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 4 above)
Comment (tncarr101) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: I am not sure because we are getting the user input for the request variable in order to allow them to set the number of comments

Comment (qirongwang) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: I'm not following. the param key "fname" can be stored as a constant right?

Comment (tncarr101) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: ohh i understand now, I was thinking you mean the httpservlet request and response variables lol
